pywb/templates/frame_insert.html

```diff
@@ -11,46 +11,33 @@
   <link rel="stylesheet" href="{{ static_prefix }}/css/bootstrap.min.css" />
   <link rel="stylesheet" href="{{ static_prefix }}/css/font-awesome.min.css">
   <link rel="stylesheet" href="{{ static_prefix }}/css/base.css">
-  <link rel="stylesheet" href="{{ static_prefix }}/css/playback.css">
 
   <script src="{{ static_prefix }}/js/jquery-latest.min.js"></script>
   <script src="{{ static_prefix }}/js/bootstrap.min.js"></script>
   <script src='{{ static_prefix }}/wb_frame.js'> </script>
 
-  {% block head %}
-  {% include 'head.html' ignore missing %}
-  {% endblock %}
-
   {% autoescape false %}
 
   {{ banner_html }}
 
+  <link rel="stylesheet" href="{{ static_prefix }}/css/playback.css">
```

Comment:
This had to move below the banner_html include to avoid overriding out settings.

Reply:
Broken record here, I know: this would be good to have as a comment in the file so we remember this going forward.